STYLE.md

```diff
@@ -315,3 +315,22 @@ you'll need to use an asynchronous-specific synchronization primitives, namely t
 itself. The documentation on `tokio`'s own
 [`Mutex`](https://docs.rs/tokio/latest/tokio/sync/struct.Mutex.html), for example, calls out the
 specifics of when and where you might need to use it vs the one from `std::sync`.
+
+
+## New Configuration Fields vs CLI flags
+
+Vector makes the distinction between configuration items that are essential to understand data
+pipelines and runtime flags that determine the details of the runtime behavior. The main configuration
+generally lives in a file in the current directory or in `/etc/vector`.
+
+Examples of main configuration fields are source, transformation, and sink declaration, as well as
+information about where any disk buffers should be persisted.
+
+For configuration items that purely inform details of Vector's runtime behavior, CLI flags without
+corresponding configuration fields should be used.
+
+An example of a runtime flag is
+`vector run --no-forced-shutdown`, which tells Vector to ignore SIGINTs and to continue running
```

Comment:
Is this a real flag? Or just a hypothetical?

Reply:
I think it was originally, but has since been renamed [--no-graceful-shutdown-limit](https://master.vector.dev/docs/reference/cli/#commands)

Reply:
fixed, thanks! https://github.com/vectordotdev/vector/pull/17784/commits/423d25210b77ea35bc1380e51d029c36d7e4a87c

Reply:
I looked in my shell history instead of looking in the docs (tactical error)